extensions/default/src/commandsModule.ts

```diff
@@ -365,6 +374,101 @@ const commandsModule = ({
       window.setTimeout(completeLayout, 0);
     },
 
+    toggleOneUp() {
+      const viewportGridState = viewportGridService.getState();
+      const { activeViewportIndex, viewports, layout } = viewportGridState;
+      const {
+        displaySetInstanceUIDs,
+        displaySetOptions,
+        viewportOptions,
+      } = viewports[activeViewportIndex];
+
+      if (layout.numCols === 1 && layout.numRows === 1) {
+        // The viewer is in one-up. Check if there is a state to restore/toggle back to.
+        const { toggleOneUpViewportGridStore } = stateSyncService.getState();
+
+        if (toggleOneUpViewportGridStore.layout) {
```

Comment:
I prefer 

```
if (!toggleOneUpViewportGridStore.layout){
  return
}
```

to reduce indentation one level

Reply:
Certainly